README.md

```diff
@@ -188,6 +190,26 @@ Some things to note:
 - `max_fee` defaults to `0`. Add `--max_fee <max_fee>` to set the maximum fee for the transaction
 - `network` defaults to the `localhost`. Add `--network <network>` to change the network for the transaction
 
+### `declare`
```

Comment:
The old `declare` doc section still exists on line 129 - should that be removed?

Reply:
I meant there are currently 2 `declare` sections.